modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-consume.adoc

```diff
@@ -0,0 +1,92 @@
+= Anypoint MQ Consume Operation
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The `consume` operation in the Anypoint MQ connector provides the ability to consume a single message at any time of the flow, from any destination, using the acknowledgment strategy that supports your use case.
+
+== Incoming Message Structure
+
+Every consumed message consists of two parts:
+
+* `payload` - Represents the value of the received message body
+* `attributes` - Contains metadata regarding the received message, such as the message properties, or its ID.
+
+image::anypoint-mq/3.x/amq-3x-consume-attributes.png[Anypoint MQ Message Attributes]
+
+
+== Consume a Message
+
+The simplest declaration to consume a message from a queue is to drag the *Consume* operation from the Mule palette to the canvas and configure the `Queue` parameter with the name of the queue from which to retrieve the message:
+
+image::anypoint-mq/3.x/amq-3x-consume-operation.png[Anypoint MQ Consume Operation]
+
+For example:
```

Comment:
Suggest ending the intro sentence with a colon and omitting this fragment.
GLOBAL for "For example: " intros